### ISSUE TITLE
Investigate why `tests/cpu-program/gfx-smoke.slang` fails on Linux when using test driver.

### ISSUE THREAD

**Issue by csyonghe** [shader-slang/slang#3176]

**Comment by swoods-nv** [comment]
Moving to Q2-- this is lower priority than everything else testing-related being worked on at the moment.

**Comment by swoods-nv** [comment]
Blocked by #3953
